types/elasticlunrjs/index.d.ts

```diff
@@ -0,0 +1,386 @@
+// Type definitions for elasticlunrjs 1.0
+// Project: https://github.com/robertIsaac/elasticlunr.js
+// Definitions by: Robert Isaac <https://github.com/robertIsaac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = elasticlunr;
```

Comment:
```suggestion
export = elasticlunr;

export as namespace elasticlunr;
```
this is most probably a proper UMD module:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library